src/styles/styles.js

```diff
@@ -2761,6 +2761,20 @@ const styles = {
         fontSize: variables.fontSizeSmall,
         ...spacing.ph2,
     },
+
+    workspaceSection: {
+        backgroundColor: colors.gray1,
```

Comment:
I wonder if we can make a more global value here that comes from the theme file... something like `themeColors.cardBackground` or something? Then in the theme file, use `gray1`

Reply:
Yes, ideally we don't want to import colors anywhere except into the theme file. I'll be working on this when I update the colors in the upcoming week, but feel free to add as a variable to`default.js` for now!

Reply:
Sounds good!